src/styles/blocks/center.scss

```diff
@@ -0,0 +1,8 @@
+.center {
```

Comment:
The class name `.center` in this SCSS file does not match the class name `class="centre"` used in the HTML file. Ensure that the class names are consistent across your HTML and SCSS files to apply the styles correctly.